index.html

```diff
@@ -349,6 +341,24 @@
     </section>
 
     <section>
+      <h2>Extensions to the <a>DocumentOrShadowRoot</a> Mixin</h2>
+
+      <dl title='partial interface DocumentOrShadowRoot' class='idl'>
+         <dt>readonly attribute Element? pointerLockElement</dt>
+         <dd>
+           <dfn title="pointerLockElement"></dfn>
+
+           <p>Returns the result of the
+           <a href="https://w3c.github.io/webcomponents/spec/shadow/#dfn-retargeting-algorithm">retargeting algorithm</a>
```

Comment:
Apparently this is the second time, if activeElement is doing something similar.

IMO it should be fine to merge this PR with the Shadow DOM reference and then it can be @annevk's responsibility to figure out the right abstraction to put in DOM and change all call sites (this and activeElement) with follow-up PRs. But maybe @annevk disagrees and would rather do everything at once.